modules/stanford_people_spotlight_views/stanford_people_spotlight_views.info

```diff
@@ -2,7 +2,7 @@ name = Stanford People Spotlight views
 description = Views for Stanford People Spotlight
 core = 7.x
 package = Stanford
-version = 7.x-2.1-alpha16
+version = 7.x-2.1-alpha17
```

Comment:
This should match across all of soe helper

Reply:
I fixed this and also fixed for the last release... looks like it was missed in the alpha16 release.